.travis/lint_06_script.sh

```diff
@@ -10,10 +10,10 @@ if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
   test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
 fi
 
-test/lint/git-subtree-check.sh src/crypto/ctaes || true
-test/lint/git-subtree-check.sh src/secp256k1 || true
-test/lint/git-subtree-check.sh src/univalue || true
-test/lint/git-subtree-check.sh src/leveldb || true
+# test/lint/git-subtree-check.sh src/crypto/ctaes
+# test/lint/git-subtree-check.sh src/secp256k1
+# test/lint/git-subtree-check.sh src/univalue
+# test/lint/git-subtree-check.sh src/leveldb
```

Comment:
We should reenable the subtree checks as well. I have seen the secp256k1 and leveldb checks fails because we removed some trailing whitespaces in some files. Fixing these issues would probably make them work again.